src/containers/admin/index.tsx

```diff
@@ -0,0 +1,312 @@
+import { ethers, providers } from "ethers";
+import { ReactChild, useContext, useEffect, useState } from "react";
+import Box from "src/components/Box";
+import LabelledDateTime from "src/components/LabelledDateTime";
+import LabelledInput from "src/components/LabelledInput";
+import { StatesContext } from "src/components/StatesContext";
+import Text from "src/components/Text";
+import useCustomContract from "src/ethereum/useCustomContract";
+import theme from "src/styleguide/theme";
+
+const Divider = () => {
+  return (
+    <Box
+      mt="1rem"
+      mb="4rem"
+      height="1px"
+      bg={`${theme.colors["purple-black"]}20`}
+    />
+  );
+};
+
+const Button = ({
+  children,
+  disabled,
+  onClick,
+  bg,
+  color,
+}: {
+  disabled?: any;
+  onClick?: any;
+  bg?: string;
+  color?: string;
+  children: React.ReactNode;
+}) => {
+  return (
+    <Box
+      as="button"
+      color={color ?? "black"}
+      cursor="pointer"
+      px="2rem"
+      py="1rem"
+      bg={bg ?? "accent-green"}
+      border="none"
+      borderRadius="4px"
+      disabled={disabled}
+      onClick={onClick}
+    >
+      {children}
+    </Box>
+  );
+};
+
+const AdminComp = ({ contractAddress }: { contractAddress: string }) => {
+  const [publicSaleTimestamp, setPublicSaleTimestamp] = useState(0);
+  const [releaseAmount, setReleaseAmount] = useState();
+  const [revealTimestamp, setRevealTimestamp] = useState(0);
+  const [projectURI, setProjectURI] = useState<String>("");
+  const [numberOfReserveTokens, setNumberOfReserveTokens] = useState();
+  const [addressToTransfer, setAddressToTransfer] = useState<String>("");
+  const [balance, setBalance] = useState("");
+  const [tokenIDToTransfer, setTokenIDToTransfer] = useState(0);
+  const [contractBalance, setContractBalance] = useState(0);
+  const [totalReleased, setTotalReleased] = useState(0);
+
+  const state = useContext(StatesContext);
+
+  const contract = useCustomContract(
+    "Collection",
+    contractAddress,
+    state.provider
+  );
+
+  useEffect(() => {
+    const getPendingPayment = async () => {
+      const res = await contract?.callStatic?.pendingPayment(
+        await state.signer.getAddress()
+      );
+      return res;
+    };
+    const getPendingPaymentInETH = async () => {
+      const balance = await getPendingPayment();
+      try {
+        const balanceInETH = ethers.utils.formatUnits(balance, 0);
```

Comment:
formatUnits(number, decimals), what it does is `number * (10^decimals)`
parseUnits(number, decimals), what it does is `number / (10^decimals)`
```suggestion
        const balanceInETH = ethers.utils.formatUnits(balance, 18);
```